react/src/components/molecules/InputRadioGroup/index.js

```diff
@@ -87,8 +91,6 @@ InputRadioGroup.propTypes = {
   defaultSelected: PropTypes.string,
   /** Whether the radio group is in error state or not. */
   error: PropTypes.bool,
-  /** Error Message content. */
-  errorMsg: PropTypes.string,
```

Comment:
why remove this prop?

Reply:
Under what condition would an error message appear for this component? I couldn't figure out when it made sense to display an error message, as you either select a value or don't.